src/pages/PageNotFound.jsx

```diff
@@ -0,0 +1,23 @@
+import { Link } from "react-router-dom";
+
```

Comment:
So we need to import the image to render it instead of passing it directly to `src`
use this 
`import logo_404 from './assets/img/page_not_found_img/error.png`